### PR TITLE
fix: Bootstrap

### PR DESCRIPTION
**Pull request by alkstal**
- Refactored the Bootstrap imports

All the bootstrap files were confusing me and according to the Bootstrap docs the setup was also incorrect, as Bootstrap variable overrides were not added correctly but in two places.

I think this change makes it easier to grasp what is going on and what needs to go where.

- Removed Bootstrap CDN

The CDN stylesheet is unnecessary as we import Bootstrap in our contexture.css file.
The CDN stylesheet imported all of Bootstrap a second time on to the page and overrid some of our styles like the font family causing Montserrat to not be used.

- Moved ui-elements style into the ui-elements HTML file

The ui-elements.scss file seems unnecessary as it only relates to the ui-elements.html file and contains very little CSS.
And since it doesn't relate to the actual application I opted to add the styles directly in the HTML file.

### CODE REVIEW

File: frontend/scss/_variables.scss [hunk 1]
```diff
@@ -1,17 +1,28 @@
-﻿@import "../node_modules/bootstrap/scss/variables";
```
Comment (alkstal) [inline frontend/scss/_variables.scss:1]: This file used to be two files.

Bootstrap says that all default variable overrides need to be done _before_ importing variables from Bootstrap.

**Comment by thijstijsma** [comment]
> * Refactored the Bootstrap imports
> 
> All the bootstrap files were confusing me and according to the Bootstrap docs the setup was also incorrect, as Bootstrap variable overrides were not added correctly but in two places.
> 
> I think this change makes it easier to grasp what is going on and what needs to go where.

Do note that with the current version of Bootstrap 5 used the colors need to be set *earlier* than what the documentation suggests, because they are used in functions as well and otherwise they will only be used in a subset of Bootstrap.

I first set it up exactly according to the docs, then ran into that.

**Comment by alkstal** [comment]
> Do note that with the current version of Bootstrap 5 used the colors need to be set _earlier_ than what the documentation suggests, because they are used in functions as well and otherwise they will only be used in a subset of Bootstrap.
> 
> I first set it up exactly according to the docs, then ran into that.

Are you sure because the `_functions.scss` file does not import any color variables? The functions don't have any dependencies at all.

**Comment by thijstijsma** [comment]
Not behind a computer right now so it might be a different file, will look into it later!

**Review by Nagelfar — APPROVED** [review]

**Pull request merged**